lbry/lbry/extras/daemon/Daemon.py

```diff
@@ -3467,15 +3471,25 @@ def jsonrpc_routing_table_get(self):
                 ]
             }
         """
-        result = await comment_client.jsonrpc_post(
-            self.conf.comment_server,
-            "get_claim_comments",
-            claim_id=claim_id,
-            parent_id=parent_id,
-            page=page,
-            page_size=page_size,
-            top_level=not include_replies
-        )
+        if hidden ^ visible:
```

Comment:
Is this supposed to be `hidden or visible`?

Reply:
It's `hidden xor visible`, since they're mutually exclusive.

Reply:
Okay, I see how it works, it's not super obvious at first.